docs/sources/administration/plugin-management/index.md

```diff
@@ -149,6 +149,19 @@ Grafana Cloud handles the plugin installation automatically.
 
 If you're logged in to Grafana Cloud when you add a plugin, log out and then log back in again to use the new plugin.
 
+
+
+### Install plugins on a self-hosted Grafana through helm
+
+In [Helm charts](https://github.com/grafana/helm-charts/blob/79a0101296b0065b18987321b3047b8c94bc1a33/charts/grafana/values.yaml#L605), you may add the plugins you wish to install as a list
+
+```
+  plugins:
+    - https://grafana.com/api/plugins/grafana-oncall-app/versions/v1.9.0/download;grafana-oncall-app # Specific version
+    - redis-datasource
+```
```

Comment:
I've tried to revise this to follow our guidelines for partial code examples: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#partial-code-examples

````suggestion
The following YAML snippet installs v1.9.0 of the Grafana OnCall App plugin and the Redis data source plugin.
You must incorporate this snippet within your Helm values file.

```yaml
plugins:
  - https://grafana.com/api/plugins/grafana-oncall-app/versions/v1.9.0/download;grafana-oncall-app
  - redis-datasource
```
````